client/rpc/src/state/state_full.rs

```diff
@@ -213,23 +213,29 @@ where
 			.map_err(client_err)
 	}
 
+	// TODO: This is horribly broken; either remove it, or make it streaming.
```

Comment:
I think the paged variant is supposed to replace this rpc, so was planed to be removed.

Reply:
Yep, I know. But technically we don't necessarily *need* to remove those non-paged variants. We could just fix them and make them stream the data. (It's out of scope of this PR though.)